### DISCUSSION TITLE
tekst labels on all bunker silo's visible..

### DISCUSSION THREAD

**Discussion by jbaarslag** [Courseplay/Courseplay_FS22#2466] [Q&A]
hello everyone, since the last update i have these teksts and red lines visible on all bunker silo's in my save game.

i tried going back to a early'er version but this did not help.

![image](https://user-images.githubusercontent.com/47272957/232276850-ceaa1d57-9a36-447b-8a4c-12550e7b1734.png)

**Comment by Tensuko** [comment]
Earlier version helps as much as load the latest version.
